src/miner.cpp

```diff
@@ -473,6 +473,8 @@ void static BitcoinMiner(CWallet *pwallet)
     );
 
     try {
+        if (solver == "tromp")
+            equi eq(1);
```

Comment:
I'm not immediately sure whether the C++ spec makes it defined or not, but I'm dubious about declaring ``eq`` in the scope of the ``if`` and then using it outside.